test/modules/sdk/objections/service.spec.unit.ts

```diff
@@ -98,4 +98,13 @@ describe("objections SDK service unit tests", () => {
 
     expect(mockAxiosRequest).toBeCalled();
   });
+
+  it("should call objections API deleting single attachment", () => {
+    objectionsSdk.deleteAttachment(COMPANY_NUMBER,
+      ACCESS_TOKEN,
+      OBJECTION_ID,
+      ATTACHMENT_ID);
+
+    expect(mockAxiosRequest).toBeCalled();
```

Comment:
we should be testing for more here, like is the correct endpoint called with the correct values. I know this is similar to other tests like getAttachments which is also lacking, so I was going to pick this up and fix them. So leave as is and I will add tests later